packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -383,7 +383,23 @@ function useAutoFocus(scopeRef: RefObject<HTMLElement[]>, autoFocus: boolean) {
       }
     }
     autoFocusRef.current = false;
-  }, []);
+  }, [scopeRef]);
```

Comment:
when does scopeRef change that isn't on mount?

Reply:
I don't think it does. This was added to clear the following ESLint warning:

> React Hook useEffect has a missing dependency: 'scopeRef'. Either include it or remove the dependency array.eslint[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)